src/controllers/BaseController.ts

```diff
@@ -65,6 +66,8 @@ export abstract class BaseController<FORM> extends BaseAsyncHttpController {
             const validationResult: ValidationResult = new SchemaValidator(this.formSchema)
                 .validate(this.httpContext.request.body);
             if (validationResult.errors.length > 0) {
+                loggerInstance()
+                    .debug(`${BaseController.name} - Validation error.`);
```

Comment:
I am not sure how useful it is. If this is just for developers then it is straightforward enough to see errors in the browser. What was your reason for adding it?

Reply:
It could be the case that the view hasn't been configured to show a validation error and hence I thought it would be useful to give the developer that feedback regardless of whether the view has been configured correctly. However, I do agree this is unnecessary since the feedback can be obtained from the status code.